tools/mkdocs_macros_bids/main.py

```diff
@@ -4,12 +4,12 @@
 Functions decorated in "define_env()" are callable throughout the
 specification and are run/rendered with the mkdocs plugin "macros".
 """
-import os
-import sys
+import os, sys
 
-schemacode_path = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
-sys.path.append(schemacode_path)
-from schemacode import macros
+code_path = os.path.abspath(os.path.join(os.path.dirname(__file__)))
+sys.path.append(code_path)
```

Comment:
Only way I found to get this to work was to start adding folders to the path, which always feels like a dirty and hacky way of doing things...

Reply:
I think without installing your packages (via `pip`), this may be the canonical way though :thinking: I think @effigies may know more about this.